aten/src/ATen/Declarations.cwrap

```diff
@@ -3885,7 +3886,7 @@
   aten_custom_call: |
     if (self_->isScalar()) {
       // Empty tensor
-      return self_->type().toScalarType(kLong).tensor({0});
+      return at::native::empty({0}, getType(toDense(self_->type().backend()), kLong));
```

Comment:
Here I ran into a weird bug where a the `._indices` of a `SparseTensor` became `SparseTensor`s themselves. I'm not sure why `self_->type().toScalarType(kLong).tensor({0});` used to get rid of the sparseness (it should return a `CUDALongType` for a `SparseCUDA<X>` type.

Reply:
If self is sparse, this is definitely insufficient to allocate a dense tensor, and if it did do that, it's a bug.